src/main/java/Parser.java

```diff
@@ -0,0 +1,293 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import exceptions.EmptyTodoException;
+import exceptions.InvalidDeadlineException;
+import exceptions.InvalidEventException;
+import exceptions.InvalidNumberException;
+import exceptions.UnknownCommandException;
+import tasks.Deadline;
+import tasks.Event;
+import tasks.Task;
+import tasks.ToDos;
+
+/**
+ * class consisting of static methods which are used to make sense of the user commands
+ */
+public class Parser {
+    /**
+     * Parse the user input and get the right response
+     * @param input String inputted by the user
+     * @param list A TaskList containing all the tasks inputted by the user
+     * @return String containing the response from duke
+     */
+    public static String getResponse(String input, TaskList list) {
+        input = input.trim();
+        try {
+            validity(input);
+        } catch (UnknownCommandException ex) {
+            return "Unknown command";
+        }
+        String[] inputs = input.split(" ");
+        String str;
+        switch(inputs[0]) {
+        case "todo":
```

Comment:
Would it be more readable and clearer if there is an empty line in between each case?